k6ext/k6test/vu.go

```diff
@@ -33,9 +33,9 @@ type VU struct {
 // ToGojaValue is a convenience method for converting any value to a goja value.
 func (v *VU) ToGojaValue(i any) goja.Value { return v.Runtime().ToValue(i) }
 
-// MoveToVUContext moves the VU to VU context, adding a predefined k6 lib State and nilling the InitEnv
-// to simulate how that is done in the real k6.
-func (v *VU) MoveToVUContext() {
+// RestoreVUState restores the state field of the VU and nilling the InitEnv field.
+// This is done to simulate how k6 operates.
+func (v *VU) RestoreVUState() {
```

Comment:
Nit picking: Calling this `Restore` is confusing to me because, for what I understand, what we are doing here is making the VU "advance" from the `Init` state to the `Active`/`Execution` state, where the `vu.State()` is populated.
Personally I would prefer calling it `InitVUState()`, but no strong opinion.

Reply:
I agree, it's confusing. Thanks for the suggestion 👍 WDYT about `ActivateVU()` with a comment saying, _"ActiveVU mimicks the activation of VU, just like in k6"_?

Reply:
I still think `InitVUState()` represents better the action that is performed in the method, but `ActivateVU()` looks better to me than `RestoreVUState()`, if you prefer that.

Reply:
Hmm, ok. I've reconsidered this. But, I concluded that it would be better to name it with the "why" instead of "how" because we can look into the method for the "how". I think I'll change it to `ActivateVU`. Thanks for the suggestion 🙇 :)

See: 48d9d7f.